scripts/scil_evaluate_bundles_individual_measures.py

```diff
@@ -118,7 +132,15 @@ def main():
                     output_measures_dict[measure_name] = []
                 output_measures_dict[measure_name].append(
                     measure_dict[measure_name])
-
+    # add set stats if there are more than one bundle
+    if len(bundles_references_tuple_extended) > 1:
```

Comment:
I like the idea of having an average across all input. I would like it for every metrics.
Average avg_len, min/max length, streamlines count, span, etc...

BUT I would add an optional boolean argument `--group_statistics` or `--average_measures`. This way if the user does not want it it can wont be compute and the output file is just like before.